content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,633 @@ keywords:
 
-1. Stop the VM if it is running, then click **Devices** to open the list of devices for the selected VM.
-2. Click on the <span class="material-icons">more_vert</span> icon at the right of the listed device you want to edit, then select **Delete**.
-   The **Delete Virtual Machine** dialog opens.
+            Select **Clone** to clone and promote a temporary snapshot of the zvol into a custom storage volume.
+            This option retains the original zvol while creating an identical copy as an instances volume.
+            Select **Move** to relocate the existing zvol to the ix-virt dataset as a volume.
 
-   {{< trueimage src="/images/SCALE/Virtualization/DeleteVirtualMachine.png" alt="Delete Virtual Machine" id="Delete Virtual Machine" >}}
+            Use this option to migrate a previously configured VM, such as after updating from TrueNAS 24.10.
+            See [Migrating Virtual Machines](https://www.truenas.com/docs/scale/25.04/gettingstarted/scalereleasenotes/#migrating-virtual-machines) from the 25.04 release notes for more information.
 
-3. Select **Delete zvol device** to confirm you want to delete the zvol device.
-   Select **Force Delete** if you want the system to force the deletion of the zvol device, even if other devices or services are using or affiliated with the zvol device.
-4. Click **Delete Device**.
+         - Click **Upload ISO** to open a file browser and select an <file>.iso</file> file from your client computer.
+            Locate your desired image file and then click **Open** to upload the file for use in instances.
 
-### Changing the Device Order
+2. Configure the **CPU & Memory** settings.
 
-1. Stop the VM if it is running, then click **Devices** to open the list of devices for the selected VM
-2. Click **Edit**.
-3. Enter the number that represents where in the boot sequence you want this device to boot in the **Devices Order** field.
-   The lower the number, the higher the device is in the boot sequence.
-4. Click **Save**.
-5. Restart the VM.
+   Compare the recommended specifications for the guest operating system with your available host system resources when allocating CPU and memory resources.
 
-### Adding a CD-ROM Device
+   {{< trueimage src="/images/SCALE/Virtualization/CreateVMCPUandMemory.png" alt="CPU & Memory - VM" id="CPU & Memory - VM" >}}
 
-Select **CD-ROM** as the **Device Type** on the **Add Device** screen and set a boot order.
+   For VMs, CPU and memory configuration is required.
 
-1. Stop the VM if it is running, then click **Devices**.
-2. Click **Add** and select **CD-ROM** from the **Device Type** dropdown list.
+   {{< include file="/static/includes/InstanceCPUMemoryProcedure.md" >}}
 
-   {{< trueimage src="/images/SCALE/Virtualization/VMAddDeviceCDROM.png" alt="Devices Add CD-ROM Type" id="Devices Add CD-ROM Type" >}}
+3. Configure the **Disks** settings to mount storage volumes for the VM.
```

Comment:
Same comment about the section header as the noun.
```suggestion
3. Configure disks settings to mount storage volumes for the VM.
```
you can also say something like this
```suggestion
3. Configure disk settings to mount storage volumes for the VM in the **Disks** section.
```